tests/plenary/capture/templates_spec.lua

```diff
@@ -3,8 +3,11 @@ local Templates = require('orgmode.capture.templates')
 describe('Capture template', function()
   local templates = Templates:new()
   it('should compile expression', function()
+    ---Backup and restore the clipboard
+    local clip_backup = vim.fn.getreg('+')
+    vim.fn.setreg('+', 'test')
     local result = templates:compile({
-      template = '* TODO\n%<%Y-%m-%d>\n%t\n%T--%T\n%<%H:%M>\n%<%A>',
+      template = '* TODO\n%<%Y-%m-%d>\n%t\n%T--%T\n%<%H:%M>\n%<%A>\n%x',
```

Comment:
Would be awesome to write a test for `%(EXP)`. I know writing tests for prompt is not really possible, but this one should work.